sce.py

```diff
@@ -0,0 +1,9 @@
+from tools.setup import SceSetupTool
+
+setup = SceSetupTool()
+setup.check_os()
+setup.check_installation("docker", "docker --version", "https://docs.docker.com/desktop/")
```

Comment:
it would be better to have two functions called `check_if_docker_installed` and `check_if_mongo_installed` to use here, sending no params.

with these two functions they can then call on `check_installation` wuhan the class with the params u use in this file.